resources/user-agents/browsers/maxthon/maxthon-4-1-to-4-3.json

```diff
@@ -8,7 +8,7 @@
     {
       "userAgent": "Maxthon #MAJORVER#.#MINORVER# for Android",
       "platform": "Android",
-      "engine": "WebKit",
```

Comment:
I swapped these as my script that I used in #1692 showed that by adding the devices I did to this file caused there to be more Blink patterns than WebKit now.